test/com/esotericsoftware/kryo/serializers/FieldSerializerGenericsTest.java

```diff
@@ -23,12 +23,15 @@
 import com.esotericsoftware.kryo.KryoTestCase;
 import com.esotericsoftware.kryo.io.Output;
 import com.esotericsoftware.kryo.serializers.FieldSerializer.FieldSerializerConfig;
+import com.esotericsoftware.kryo.serializers.FieldSerializerTest.HasGenerics;
+import com.esotericsoftware.kryo.serializers.FieldSerializerTest.ListContainer;
```

Comment:
These two classes are now only used in `FieldSerializerGenericsTest`. Please move them to this class.